tests/Php84/Php84Test.php

```diff
@@ -64,6 +64,22 @@ public function testArrayAll(array $array, callable $callback, bool $expected)
         $this->assertSame($expected, array_all($array, $callback));
     }
 
+    public function testCurlHttp3Constants() {
+        if (!function_exists('curl_version')) {
+            $this->markTestSkipped('Curl extension is not available.');
+        }
+
+        // If the curl_version()['features'] bitmask contains the bits for CURL_VERSION_HTTP3,
+        // it means Curl supports HTTP3.
+        if (!(curl_version()['features'] & 33554432)) { // 33554432 = CURL_VERSION_HTTP3
```

Comment:
why is this check needed?

Reply:
per https://github.com/php/php-src/blob/3ed884fab73c5807fe2b3001ee79545c27ad5e70/ext/curl/curl_arginfo.h#L854-L856
we'd should rather check the version of curl in addition to checking if the ext is installed?